package Lesson104_ExceptChapter.exception;

public interface IntfWithEx {
    void danger() throws Exception;

    void safe();
}
